prime-router/metadata/fhir_mapping/hl7/datatype/Identifier_SystemID.yml

```diff
@@ -1,54 +1,54 @@
 type_1:
-   condition: $identifierCoding NOT_NULL
-   valueOf: datatype/CodeableConcept
-   expressionType: resource
-   specs: CX.5 | $typeSegment
-   vars: 
-     # Because calculated system id's don't have text if there is a valid system, 
-     # see if there is a valid code and system as input to creating text in the CodeableConcept
-     identifierCoding: CODING_SYSTEM_V2_IDENTIFIER, CX.5 | $typeSegment
+  condition: $identifierCoding NOT_NULL
+  valueOf: datatype/CodeableConcept
+  expressionType: resource
+  specs: CX.5 | $typeSegment
+  vars:
+    # Because calculated system id's don't have text if there is a valid system,
+    # see if there is a valid code and system as input to creating text in the CodeableConcept
+    identifierCoding: CODING_SYSTEM_V2_IDENTIFIER, CX.5 | $typeSegment
 
 type_2: # used in instance where objects are passed directly to the codeable concept instead of coming directly from a field
   valueOf: datatype/CodeableConcept_var
   generateList: true
   expressionType: resource
 
-system_1: 
-     condition: $systemExists NOT_NULL
-     type: SYSTEM_ID
```

Comment:
When using SYSTEM_ID, the string automatically gets a prefix (I think of `urn:id:`) and has " " replaced with "_", so using a string helps this be simply preserved through translation.